### PR TITLE
Remove RVM dependency for integration specs

### PR DESCRIPTION
**Pull request by jowl**
This PR suggests enabling non-RVM users, such as myself, to run the integration specs.

### CODE REVIEW

File: spec/integration/integration_spec.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@
 describe 'Packaging and running a project' do
   def isolated_run(dir, cmd)
     Dir.chdir(dir) do
-      Bundler.clean_system("PATH=$GEM_HOME/bin:$PWD/../../../tmp/hadoop-current/bin:$PATH HUMBOLDT_TEST_PROJECT_PATH=$PWD rvm $RUBY_VERSION@humboldt-test_project do #{cmd}")
+      Bundler.clean_system(cmd)
```
Comment (grddev) [inline spec/integration/integration_spec.rb]: I still think you need to prepend Hadoop to the path, and the Travis runs seem to agree with me.

Comment (jowl) [inline spec/integration/integration_spec.rb]: Cheers, I didn't notice that part of the path and it worked nicely on my machine. Should work now.

**Comment by jowl** [comment]
@grddev could you please give this another quick review before I merge?

**Comment by grddev** [comment]
It looks very clean to me. It obviously works with the RVM setup on Travis, and if you say it works with your non-RVM setup locally, I say go ahead.

**Pull request merged**